manifests/plugin/shellhooks.pp

```diff
@@ -0,0 +1,30 @@
+# = Foreman Proxy Shelhooks plugin
```

Comment:
```suggestion
# = Foreman Proxy Shellhooks plugin
```

Reply:
That's like the 7th place where I made that typo -.-